unitTests/ui/DynamicFacet/DynamicFacetTest.ts

```diff
@@ -35,9 +35,13 @@ export function DynamicFacetTest() {
 
       spyOn(test.cmp.values, 'clearAll').and.callThrough();
       spyOn(test.cmp.values, 'render').and.callThrough();
+      spyOn(test.cmp.values, 'createFromResponse');
```

Comment:
Not really in scope for this PR, but I think that when you need to pretty much "mock everything on a given object and pass through", there must be a better way to do that using jasmine. 

Since it's JS and everything is mutable, you could have a proxy constructor or something ?

Reply:
Noted, I'll try to do something about it